src/aura/STG_CMP_Addr/STG_CMP_Addr.cmp

```diff
@@ -78,7 +81,74 @@
             <br />
             <ui:outputText aura:id="cleanUpMsg" value="{!$Label.c.stgCleanupQueuedEmailSent}" class="slds-text-color--weak slds-hide" />
         </div>
+		<!-- End of Preferred Email -->
+        
+        <!--  Preferred Phone -->
```

Comment:
Is Julie aware of these changes? Just want to make sure the docs team knows there are UI changes and that we will want a separate WI to document this new functionality if we don't already have one.

I see W-034361, but I don't know if that just applies to the TDTM documentation or additional UI documentation as well.

Reply:
I think so, @juliesnow just in case tagging you.